configure

```diff
@@ -68,6 +68,7 @@ Usage: $0 [OPTION]... [VAR=VALUE]...
     --disable-btest        don't install BTest
     --disable-btest-pcaps  don't install Zeek's BTest input pcaps
     --disable-cpp-tests    don't build Zeek's C++ unit tests
+    --disable-port-prealloc disable pre-allocating the PortVal array in ValManagerx
```

Comment:
```suggestion
    --disable-port-prealloc disable pre-allocating the PortVal array in ValManager
```

I could see us not even hooking it up with `./configure` and assuming the advanced `-D` cmake settings if someone wants to flip it.